packages/run-protocol/test/vaultFactory/test-liquidator.js

```diff
@@ -342,19 +342,73 @@ const makeDriver = async (t, initialPrice, priceBase) => {
     E(lender).getCollateralManager(aethBrand),
   ).getNotifier();
   let managerNotification = await E(managerNotifier).getUpdateSince();
-  /** @type {UserSeat<VaultKit>} */
-  let vaultSeat;
-  /** @type {VaultKit} */
-  let vault;
+
   /** @type {UserSeat<VaultKit>} */
   let lastSeat;
-  let notifier;
   let notification = {};
   let lastOfferResult;
+  const makeVaultDriver = async (collateral, debt) => {
+    /** @type {UserSeat<VaultKit>} */
+    const vaultSeat = await E(zoe).offer(
+      await E(lender).makeVaultInvitation(),
+      harden({
+        give: { Collateral: collateral },
+        want: { RUN: debt },
+      }),
+      harden({
+        Collateral: aethMint.mintPayment(collateral),
+      }),
+    );
+    const {
+      vault,
+      publicNotifiers: { vault: notifier },
+    } = await E(vaultSeat).getOfferResult();
+    t.truthy(await E(vaultSeat).hasExited());
```

Comment:
it should return a boolean
```suggestion
    t.true(await E(vaultSeat).hasExited());
```